cycode/cli/utils/string_utils.py

```diff
@@ -47,3 +47,13 @@ def generate_random_string(string_len: int):
 
 def get_position_in_line(text: str, position: int) -> int:
     return position - text.rfind('\n', 0, position) - 1
+
+
+@staticmethod
```

Comment:
it is not a class member anymore. we should remove "@staticmethod"

Reply:
Fixed